src-web/nls/platform.properties

```diff
@@ -170,6 +170,7 @@ modal.ansible.scheduleMode.once.title = Run once mode
 modal.ansible.scheduleMode.once.info = When this policy is violated, the automation will run one time. Once it runs, it will be set to disabled.
 modal.ansible.scheduleMode.disabled.title = Disable automation
 modal.ansible.scheduleMode.disabled.info = Automation won't run until you change this setting.
+modal.ansible.unsaved.data = You have made the change on configuration, do you want to save your policy automation? Or re-click 'Cancel' button to close.  
```

Comment:
@dockerymick can you review?

Reply:
I recommend saying, "You made changes to the configuration, do you want to save your policy automation. Click the Cancel button to close"

Can you italicize Cancel or bold it instead of the single quotes?

Reply:
Sure